ddtrace/tracer/textmap.go

```diff
@@ -177,6 +177,9 @@ func getPropagators(cfg *PropagatorConfig, env string) []Propagator {
 	if ps == "" {
 		return defaultPs
 	}
+	if ps == "none" {
```

Comment:
Looking at like 197 here it seems like setting a value of `none,b3` would result in a message "unrecognized propagator: none". Can we add a special case for `none` that instead warns the user something like `Propagator "none" has no effect when combined with other propagators. To disable propagator set only "none"` (Feel free to change the message). (And a quick little unit test update to make sure that future users don't accidentally break that message)

Reply:
Good point 👍 Updated